src/html.pre.js

```diff
@@ -55,9 +56,29 @@ function pre(context) {
   }
 
   // ensure content.data is present
-  if (!context.content.data) {
-    context.content.data = {};
-  }
+  content.data = content.data || {};
+
+  // extract metadata
+  const { meta = {} } = content;
+  // description: text from paragraphs with 10 or more words
+  const desc = $sections
+    .filter('.default')
+    .find('p')
+    .filter(function minLength() {
+      return $(this).text().split(' ').length >= 10;
```

Comment:
I would use white space here:

```suggestion
      return $(this).text().split(/\s+/g).length >= 10;
```

Reply:
I don't think the global flag is required, though.